front/deploygroup.form.php

```diff
@@ -65,8 +65,8 @@
       $values['plugin_fusioninventory_deploygroups_id'] = $_GET['id'];
       $group_item->add($values);
    } else {
-      $item = getAllDatasFromTable($group_item->getTable(),
-                                   "plugin_fusioninventory_deploygroups_id='".$_GET['id']."'");
+      $item = $dbUtils->getAllDataFromTable($group_item->getTable(),
```

Comment:
Should be `['plugin_fusioninventory_deploygroups_id' => $_GET['id']]`.

Same for all other occurrences.

Reply:
This one is still incorrect

Reply:
Done amending commits